Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -75,6 +75,11 @@ class AppSettingsTableViewController: SettingsTableViewController {
             ]
         }
 
+        generalSettings +=  [
```

Comment:
I wanted to keep it at the end of the list, i.e. even after the platform-dependent "Use Compact Tabs".
But I see that it would actually make sense to keep it together with the always-present other general settings. So I'll move the line up as requested. Thank you for the suggestion.